docs_input/api/linalg/other/toeplitz.rst

```diff
@@ -0,0 +1,33 @@
+.. _toeplitz_func:
+
+toeplitz
+========
+
+Generate a toeplitz matrix
+
+`c` represents the first column of the matrix while `r` represents the first row. `c` and `r` must
+have the same first value; if they don't match, the first value from `r` will be used.
```

Comment:
We document here that if c and r don't have a first value that match we use r[0], however looking at the code, it looks like we always use op1[0] for the first element, which maps to c param from the user API.

Reply:
(this is because we use the conditional j>i). We could change the conditional or change the wording here to reflect is always used for (0,0) and its diagonal.

Reply:
Yes, I updated the code after writing that. Thanks